packages/widgets/src/blocks/widget-group/index.php

```diff
@@ -70,6 +75,8 @@ function note_sidebar_being_rendered( $index ) {
 /**
  * Clear whatever we set in note_sidebar_being_rendered() after WordPress
  * finishes rendering a sidebar.
+ *
+ * @global array $_sidebar_being_rendered
```

Comment:
```suggestion
 * @global int|string $_sidebar_being_rendered
```

Reply:
@t-hamano I have Updated all the changes as per suggested.

Reply:
Thanks for the update!

Finally, could you just commit to [this comment suggestion](https://github.com/WordPress/gutenberg/pull/59931/files#r1531458457)? I think everything else is fine 👍

Reply:
Updated this change in latest commit.